TSP/TSP.API/Controllers/EmployeeController.cs

```diff
@@ -9,16 +9,16 @@
 
 namespace TSP.API.Controllers
 {
-    public class EmployeeController : ControllerBase
+   /* public class EmployeeController : ControllerBase
```

Comment:
This commented code mustn't reach a PR.

Reply:
if i uncomment there will be a build error

Reply:
Fix the error

Reply:
I'm trying to fix it

Reply:
fix